src/wait-strategy.ts

```diff
@@ -74,7 +74,9 @@ export class LogWaitStrategy extends AbstractWaitStrategy {
 
   public async waitUntilReady(container: Dockerode.Container): Promise<void> {
     log.debug(`Waiting for log message "${this.message}"`);
-    const stream = await containerLogs(container);
+    const inspect = await inspectContainer(container);
+    const startedAt = inspect.state.startedAt.getTime();
+    const stream = await containerLogs(container, { since: Math.floor(startedAt / 1000), timestamps: true });
```

Comment:
Is the rounding still necessary?

Reply:
Unfortunately, yes, as without it, we don't get back all logs that might be necessary (i.e one of the startup strategy tests fails)